Sources/StreamChatUI/CommonViews/Suggestions/ChatMentionSuggestionView/ChatMentionSuggestionView_Tests.swift

```diff
@@ -50,15 +50,15 @@ class ChatMentionSuggestionView_Tests: XCTestCase {
         let view = ChatMentionSuggestionView().withoutAutoresizingMaskConstraints
         view.components = .mock
         view.widthAnchor.constraint(equalToConstant: Self.defaultCellWidth).isActive = true
-
+        
+        view.content = chatUserNoName
+        AssertSnapshot(view, suffix: "user name not set")
+        
```

Comment:
Can we create a new testing case to make sure this issue doesn't comeback?

Reply:
✅ Added the test